artifacts/definitions/Windows/Detection/TemplateInjection.yaml

```diff
@@ -0,0 +1,108 @@
+name: Windows.Detection.TemplateInjection
+description: |
+    This content will detect injected templates in Office documents.   
+    
+    Template injection is a form of defence evasion where a malicious macro 
+    is loaded into an OOXML document via a resource file masquerading as 
+    an office template.   
+
+author: Matt Green - @mgreen27
+
+reference:
+  - https://attack.mitre.org/techniques/T1221/
+  - https://www.sans.org/reading-room/whitepapers/testing/template-injection-attacks-bypassing-security-controls-living-land-38780
+
+parameters:
+  - name: SearchGlob
+    description: Glob to search
+    default: C:\Users\**\*.{docx,dotx,xlsx,xltx,pptx,potx,ppsx,sldx}
+  - name: TemplateRegex
+    description: Regex to search inside resource section.
+    default: '^(https?|smb|\\\\\\\\)'
+  - name: UploadDocument
+    type: bool
+    description: Select to upload document on detection.
+  
+sources:
+  - precondition: SELECT OS From info() where OS = 'windows'
+  
+    queries: 
```

Comment:
Maybe use `query: |`?

Reply:
just added this in. The base for this artefact was previously queries and wasnt changed